debian/collectd-core.collectd.default

```diff
@@ -12,6 +12,10 @@ USE_COLLECTDMON=1
 # default: 30
 MAXWAIT=30
 
+# try a terminal kill -9 if the process refuses to shut down
+# default: 0
+declare -i USE_SIGKILL=1
```

Comment:
This is a very generic name, missing the context of where this would be applied. Can you think of something better? I don't have a good idea myself yet but something along the lines of KILL_AFTER_WAIT or similar.